core-blocks/block/index.js

```diff
@@ -173,7 +173,7 @@ export const name = 'core/block';
 export const settings = {
 	title: __( 'Shared Block' ),
 	category: 'shared',
-	isPrivate: true,
+	allowedParents: false,
```

Comment:
Update: I've seen why this allows `true` and `false` types but I think they make the API unclear. I'd rather constants that signified the meaning of what `true` and `false` are achieving here instead of just using those values without info.